scripts/updates/api/dump-repology-data

```diff
@@ -24,16 +24,17 @@
 
 
 from requests import get as requests_get
+from time import sleep as sleep
 
 
 def get_repology_data(last_project, project):
     url = f"https://repology.org/api/v1"
     if project != "":
         url = f"{url}/projects/{project}?inrepo=termux&outdated=1"
```

Comment:
I think same fix should be applied here (add "/"). Usage in python-apt and texinfo auto update.

Reply:
Done.